pkg/util/test/util.go

```diff
@@ -643,6 +643,74 @@ func validateImageTag(tag, namespace string, listOptions map[string]string) erro
 	return nil
 }
 
+func validateMonitoring(cluster *corev1alpha1.StorageCluster, timeout, interval time.Duration) error {
+	if cluster.Spec.Monitoring != nil && *cluster.Spec.Monitoring.EnableMetrics {
+		monitoringDeployments := []string{"prometheus-operator", "grafana"}
+		monitoringStatefulSets := []string{"alertmanager-portworx", "prometheus-prometheus"}
```

Comment:
Currently we don't deploy alert manager and grafana. So let's remove the check for those deployments/statefulsets.